src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/FinanceTrackerParser.java

```diff
@@ -157,6 +158,10 @@ public Command parseCommand(String userInput, UiState uiState) throws ParseExcep
                         Tab.OVERVIEW, Tab.EXPENSES, Tab.INCOME));
             }
 
+        case SetExpenseLimitCommand.COMMAND_WORD:
+        case SetExpenseLimitCommand.COMMAND_ALIAS:
+            return new SetExpenseLimitCommand(arguments.trim());
```

Comment:
Uncaught exception when no argument is supplied i.e. `set-expense-limit`.

![image](https://user-images.githubusercontent.com/53135010/96952790-3cea6e00-1522-11eb-8d1d-561f9305a73d.png)

This could be handled either here in the parser or within the command execution (as the amount is being validated).